cassandra/tablets.py

```diff
@@ -51,7 +58,42 @@ def get_tablet_for_key(self, keyspace, table, t):
             return tablet[id]
         return None
 
+    def drop_tablet(self, keyspace: str, table: str = None):
+        print("drop tablets for ", keyspace, table)
+        with self._lock:
+            if table is not None:
+                self._tablets.pop((keyspace, table), None)
```

Comment:
🔧 Please use logging instead of prints. There are more occurences of that in the PR, I won't specifically mark them.

Reply:
It is just leftovers, removed them